test/unit/configuration-test.js

```diff
@@ -171,405 +171,226 @@ describe('configuration.applyLoggingOptions()', () => {
 
 describe('configuration._coerceRemovedOptions()', () => {
```

Comment:
The function doesn't exist anymore, right? Should we rename the tests?

Reply:
Yes. I'm planning to add unit tests for those functional utils which now compose configuration handling. I have a few of them unstaged, will push once the pull request is cleaned and your comments are taken into account.

Reply:
It makes sense to keep the test so we know we didn't break anything with the refactor, but we should still rename the test, perhaps at least to just "configuration" as it basically tests the "configuration" folder now.